kolibri/plugins/learn/assets/src/vue/side-nav/index.vue

```diff
@@ -68,68 +66,58 @@
   @require '../learn'
   @require 'jeet'
 
-  $nav-element-height = 150px
   $font-size = 1em
+  $nav-size = $left-margin - $card-gutter * 0.5
+  $portrait-scale = 0.8
 
   nav
     background: $core-bg-light
-    font-size: $font-size
     font-weight: 300
     overflow: hidden
     position: fixed
     z-index: 2
-    @media screen and (orientation: landscape)
+    @media screen and (min-width: $portrait-breakpoint + 1)
+      font-size: 1em
       height: 100%
       top: 0
       left: 0
-      width: $left-margin - $card-gutter * 0.5
-    @media screen and (orientation: portrait)
+      width: $nav-size px
+    @media screen and (max-width: $portrait-breakpoint)
+      font-size: $font-size * $portrait-scale
+      height: $nav-size * $portrait-scale px
       bottom: 0
       width: 100%
-      height: auto
-      display: table
-      font-size: 10px
-      
-  .nav-row
-    @media screen and (orientation: portrait)
-      display: table-row
-
-  .nav-spacer
-    height: 0
-    @media screen and (orientation: portrait)
-      height: 40px
+
+  .content
+    align(all)
 
   a
-    text-align: center
-    position: relative
-    height: $nav-element-height
-    display: block
     margin: 0
     padding: 0
-    @media screen and (orientation: portrait)
-      display: table-cell
-      height: auto
+    position: relative
+    @media screen and (min-width: $portrait-breakpoint + 1)
+      height: 150px
+      display: block
+    @media screen and (max-width: $portrait-breakpoint)
+      height: 100%
+      column(1/2)
```

Comment:
it would be nice to not hard coded the number of tabs. But we can probably work around this with vue.js dynamic class assignment.